egs/ami/s5/run_prepare_shared.sh

```diff
@@ -3,41 +3,15 @@
 . ./cmd.sh
 . ./path.sh
 
-# To run this script you need SRILM,
 
 # Path to Fisher transcripts LM interpolation (if not defined only AMI transcript LM is built),
-FISHER_TRANS=`pwd`/eddie_data/lm/data/fisher/part1 # Edinburgh, [DEFAULT]
-# Path where AMI gets downloaded (or where locally available),
-AMI_DIR=$PWD/DOWNLOAD/amicorpus # [DEFAULT]
+#FISHER_TRANS=`pwd`/eddie_data/lm/data/fisher/part1 # Edinburgh,
+#FISHER_TRANS=/export/corpora4/ami/fisher_trans/part1 # JHU,
+#FISHER_TRANS=/mnt/matylda2/data/FISHER/fe_03_p1_tran # BUT,
+FISHER_TRANS=/export/ws15-ffs-data/corpora/LDC/LDC2004T19/fe_03_p1_tran # JSALT2015 workshop, cluster AWS-EC2,
 
```

Comment:
Hi, I don't understand what is the problem here... Sure the locations are different. AMI_DIR is no longer used in run_prepare_shared.sh and the FISHER_TRANS in Edinburg is replaced by JSALT default. Well I am not sure what should be the default for FISHER_TRANS, it could be BUT location. if nobody objects :)